### ISSUE TITLE
React - Todo Template initial Build error

### ISSUE THREAD

**Issue by joeJazdzewski** [FountainJS/generator-fountain-webpack#71]
I created the project template for the To Do example. When built the example, I got "Error in ./src/app/components/TodoTextInput.js 40:13 Unnecessarily quoted property 'edit'".

**Comment by Swiip** [comment]
Yep, we had some regression after the XO update. The tech template is better tested so I'm not so surprised we missed something to Todo. We have to check initial linting on Todo and Hello templates...
